Lib/ldapurl.py

```diff
@@ -130,14 +130,14 @@ def __ne__(self,other):
     return not self.__eq__(other)
 
 
-class LDAPUrlExtensions(UserDict):
+class LDAPUrlExtensions(IterableUserDict):
```

Comment:
I'd rather use `MutableMapping` than `IterableUserDict`, for reasons I give in https://github.com/python-ldap/python-ldap/pull/186.

Reply:
I've reverted this change to defer it to the linked PR.